catalogData/elasticsearch24/3x-ha/k8s/statefulset-data.json

```diff
@@ -151,7 +151,7 @@
                 }
               }
             },
-            "imagePullPolicy": "Always"
+            "imagePullPolicy": "IfNotPresent"
```

Comment:
I think so, but doing that right is out of scope for this story IMO.

I think until we can get to the promote images via tests feature, we could ship a cronjob on the minion nodes that docker pulls our images on some interval, WDYT?